buildscripts/incremental/test.sh

```diff
@@ -41,7 +41,34 @@ else
     esac
 fi
 
+# If TEST_THREADING is set in the env, then check that Numba agrees that the
+# environment can support the requested threading.
+function check_sysinfo() {
+    cmd="import os;\
+         from numba.misc.numba_sysinfo import get_sysinfo;\
+         print(get_sysinfo()['$1 Threading'])"
+    python -c "$cmd"
```

Comment:
Would an `eval` on the result of the look and followed by an `assert` in the string command be as clear as the additional "check result of query" in bash? Would save having to store the print into a bash var and test it later.

Reply:
an uncaught exception should cause the python process to exit `1`. Or you can also call `os.exit(1)`.

Reply:
Indeed, I'll give this a try to limit mixing two languages.

Reply:
@stuartarchibald, do you want to make this change?

Reply:
Done in 1b4666b. Now uses `assert`.